devel/ci/integration/tests/fixtures/rabbitmq.py

```diff
@@ -0,0 +1,70 @@
+# Copyright © 2018 Red Hat, Inc.
+#
+# This file is part of Bodhi.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+import time
+
+import pytest
+
+
+@pytest.fixture(scope="session")
+def rabbitmq_container(docker_backend, docker_network):
+    """Fixture preparing and yielding a RabbitMQ container.
+
+    Args:
+        docker_backend (conu.DockerBackend): The Docker backend (fixture).
+        docker_network (str): The Docker network ID (fixture).
+
+    Yields:
+        conu.DockerContainer: The RabbitMQ container.
```

Comment:
For new code, I'd like us to start using type annotations instead of docblocks to mark the types of things. So the above would be like this:

```
def rabbitmq_container(docker_backend: conu.DockerBackend, docker_network: str) -> conu.DockerContainer:
    """
    Fixture preparing and yielding a RabbitMQ container.

    Args:
        docker_backend: The Docker backend (fixture).
        docker_network: The Docker network ID (fixture).
    Yields:
        The RabbitMQ container.
```